### PR TITLE
[docs] Add a readme for prom-metrics-linter

### PR DESCRIPTION
**Pull request by avlitman**
Add a README file for prom-metrics-linter.

### CODE REVIEW

**Comment by avlitman** [comment]
Hi @sradco, I will appreciate your review.

**Comment by avlitman** [comment]
Hi @sradco, I will appreciate your review.

File: test/metrics/prom-metrics-linter/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# CI Metrics Tests
+## prom-metrics-linter test
+### Test
+prom-metrics-linter is a common image for testing metrics follow the naming conventions in Kubevirt sub-operators projects.
```
Comment (sradco) [inline test/metrics/prom-metrics-linter/README.md:4]: prom-metrics-linter is a common image for testing metrics follow the naming conventions in Kubevirt sub-operators projects. ->
prom-metrics-linter is a common image for testing that the metric names follow the naming conventions in all the Kubevirt sub-operators projects.

File: test/metrics/prom-metrics-linter/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+This linter is based on [promlint](https://github.com/prometheus/client_golang/blob/v1.16.0/prometheus/testutil/promlint/promlint.go), with additional tests customized for Kubevirt.
+### Test tools
+The test is using GitHub Actions to create a docker image with the linter tests.
+The Dockerfile is here: [Dockerfile](https://github.com/kubevirt/monitoring/blob/main/test/metrics/prom-metrics-linter/Dockerfile).\
+The action is triggered by version release, here is the [GitHub action](https://github.com/kubevirt/monitoring/blob/main/.github/workflows/prom-metrics-linter.yaml), that build the image and pushes it to quay.io, with a new tag.
```
Comment (sradco) [inline test/metrics/prom-metrics-linter/README.md:9]: that build the image -> that builds the image

File: test/metrics/prom-metrics-linter/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+The assumption is that there won't be frequent changes in the prom-metrics-linter image.
+
+### Run the test
+The linter image require 3 parameters, the operator name, the sub-operator name, and a json file with the sub-operator metrics.\
```
Comment (sradco) [inline test/metrics/prom-metrics-linter/README.md:14]: require > requires

File: test/metrics/prom-metrics-linter/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+For collecting the metrics in the right format, use common help functions: [pkg/metrics/parser](https://github.com/kubevirt/monitoring/tree/main/pkg/metrics/parser).\
```
Comment (sradco) [inline test/metrics/prom-metrics-linter/README.md:15]: Perhaps?
For collecting the metrics in the right format, **we** use **this** common help functions

File: test/metrics/prom-metrics-linter/README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+Run the linter image:\
```
Comment (sradco) [inline test/metrics/prom-metrics-linter/README.md:17]: **Then r**un the linter image:

**Comment by avlitman** [comment]
@sradco Fixed all your comments.

File: test/metrics/prom-metrics-linter/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+The assumption is that there won't be frequent changes in the prom-metrics-linter image.
+
+### Run the test
+The linter image requires 3 parameters, the operator name, the sub-operator name, and a json file with the sub-operator metrics.\
+For collecting the metrics in the right format, please use this common help functions: [pkg/metrics/parser](https://github.com/kubevirt/monitoring/tree/main/pkg/metrics/parser).
+
+Then run the linter image:\
+`<container runtime> run -i "quay.io/kubevirt/prom-metrics-linter:<linter_image_tag>" --metric-families=<metrics_json_format>" --operator-name=kubevirt --sub-operator-name=<sub-operator-name>`.\
+Note: if there is no sub operator, set sub-operator-name to the operator-name.
```
Comment (sradco) [inline test/metrics/prom-metrics-linter/README.md:19]: @avlitman  Note: **If** there is no sub operator, set sub-operator-name **the same as** the operator-name.

Comment (avlitman) [reply test/metrics/prom-metrics-linter/README.md]: @sradco Fixed

**Review by sradco — APPROVED** [review]

**Pull request merged**